### PR TITLE
[ENH] GenialisExpression: add quantile transform 

### LINKED ISSUE #257: Gene expression normalization
I gathered my thoughts on gene expression normalization. Please support the following normalization methods (and use given labels):

- Quantile normalization
- Log2(x + 1)
- Z-score (row-wise) 
- Z-score (column-wise)
- Quantile transform to normal (row-wise)
- Quantile transform to normal (column-wise)
- Quantile transform to uniform (row-wise)
- Quantile transform to uniform (column-wise)

Quantile normalization should be implemented as defined on the [wiki page](https://en.wikipedia.org/wiki/Quantile_normalization). I did not find an implementation in Python yet, but the approach is commonly used for the normalization of gene expression data in R. Search for a Python implementation. If you don't find one, implement it accordingly.

For quantile transforms use [quantile_transform](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.quantile_transform.html) method from sklearn. Set argument `n_quantiles=min(X.shape[0], 100)` for row-wise and `n_quantiles=min(X.shape[1], 100)` for column-wise. 

The methods are listed in order as they should be applied if multiple are selected. I am aware not all combinations make sense, but let's leave this to the discretion of the user. Log2 should be selected by default.

_Originally posted by @mstajdohar in https://github.com/biolab/orange3-bioinformatics/issues/256#issuecomment-662316713_

### PR DESCRIPTION
**Pull request by JakaKokosar**
##### Issue
Fixes #257 

##### Description of changes

normalizations

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by JakaKokosar** [comment]
@Martin-Spendl I've added Quantile Normalization
